src/server/common/model/answer/radio-button/radio-button.js

```diff
@@ -34,7 +34,9 @@ const createRadioSchema = (config) => {
  */
 const handleConfig = (context, config) => {
   if (typeof config === 'function') {
-    return config(context ?? {})
+    return config(
+      context ?? /** @type {RawApplicationState} */ ({ application: {} })
```

Comment:
`RawApplicationState` shouldn't have `application` nested within it.  It should look like it looked before now, e.g.:

```javascript
{
  origin: { ... }
}
```